### ISSUE TITLE
Verify that constants exist

### ISSUE THREAD

**Issue by ruudk** [VincentLanglet/Twig-CS-Fixer#248]
It's possible to use the `constant('App\\SomeClass::SomeConstant')` function in to reference a constant. 

But it will only be verified on runtime.

It would be great if we could define a non-fixable rule that verifies these when doing a lint.

Now I'm not sure if this is something that should be part of Twig CS Fixer, but if it is, what would be the best way to do this? I feel like creating a NodeVisitor that finds these `constant` functions and then reports errors when the constant does not exist would be best. Otherwise I have to check this based on the tokens.

**Comment by VincentLanglet** [comment]
I agree I'm not sure this lib is the best one for such rule.
It's more a "static-analysis" rule rather than a "linter" one.

**Comment by ruudk** [comment]
This can now be implemented using https://github.com/VincentLanglet/Twig-CS-Fixer/issues/249 🥳

**Comment by VincentLanglet** [comment]
Yes, thanks for your work.

I'll try a small refacto first https://github.com/VincentLanglet/Twig-CS-Fixer/pull/251 (WIP)

**Comment by ruudk** [comment]
Yes, I have it ready on our project. Will contribute it back next week.
